### PR TITLE
Loop command

### PR DESCRIPTION
**Pull request by JamesHawkinss**
# Pull request

 - [x] I have read and agreed to the [code of conduct](https://github.com/CascadeBot/CascadeBot/blob/master/.github/CODE_OF_CONDUCT.md).
 - [x] I have read and complied with the [contributing guidelines](https://github.com/CascadeBot/CascadeBot/blob/master/.github/CONTRIBUTING.md).
 - [x] What I'm implementing was assigned to me or was not being worked on and is a somewhat requested feature. For reference see our [GitHub projects](https://github.com/orgs/CascadeBot/projects).
 - [x] I have tested all of my changes.
 
## Added/Changed
What did you add and/or change for this.

## Feature
If this is a new feature, why should we add it to the bot.

### CODE REVIEW

**Comment by JamesHawkinss** [comment]
> For loop we should have no args go through the loop modes (Off -> Playlist -> Song) or add buttons for them

i gotchu

**Comment by JamesHawkinss** [comment]
@weeryan17 i dunno how i'd compare the current loop mode and like loopMode(DISABLED)

**Comment by binaryoverload** [comment]
This PR needs completing, if you need help please ask in discord.

File: src/main/java/org/cascadebot/cascadebot/commands/music/LoopCommand.java [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+package org.cascadebot.cascadebot.commands.music;
+
+import net.dv8tion.jda.core.entities.Member;
+import org.cascadebot.cascadebot.commandmeta.*;
+import org.cascadebot.cascadebot.commandmeta.Module;
+import org.cascadebot.cascadebot.permissions.CascadePermission;
+
+import java.util.Set;
+
+import static org.cascadebot.cascadebot.music.CascadePlayer.LoopMode.*;
```
Comment (weeryan17) [inline src/main/java/org/cascadebot/cascadebot/commands/music/LoopCommand.java]: I know binary is fixing this, but WHY ARE YOU STILL USING STATIC!

Comment (JamesHawkinss) [reply src/main/java/org/cascadebot/cascadebot/commands/music/LoopCommand.java]: should i just use import instead?

**Pull request merged**